doc/api/test.md

```diff
@@ -502,14 +502,8 @@ used as an in depth coverage report.
 node --test --experimental-test-coverage --test-reporter=lcov --test-reporter-destination=lcov.info
 ```
 
-### Limitations
```

Comment:
I think we should leave this subtitle.
It is a keyword to search for, and also a separation that will make this easier to find.